packages/amplify-console-hosting/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can you please take a look at the package json's of other packages in this repo and replicate the scripts?

Reply:
Will change